ios-11-0-dep-9-0-bitcode.cmake

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2017, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_IOS_11_0_DEP_9_0_BITCODE_CMAKE_)
+  return()
+else()
+  set(POLLY_IOS_11_0_DEP_9_0_BITCODE_CMAKE_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_clear_environment_variables.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+set(IOS_SDK_VERSION 11.0)
+set(IOS_DEPLOYMENT_SDK_VERSION 9.0)
+
+set(POLLY_XCODE_COMPILER "clang")
+polly_init(
+    "iOS ${IOS_SDK_VERSION} / Deployment ${IOS_DEPLOYMENT_SDK_VERSION} / Universal (iphoneos + iphonesimulator) / \
+${POLLY_XCODE_COMPILER} / \
+bitcode / \
+c++11 support"
+    "Xcode"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
+
+include(polly_fatal_error)
+
+# Fix try_compile
+include(polly_ios_bundle_identifier)
+
+set(CMAKE_MACOSX_BUNDLE YES)
+set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "iPhone Developer")
+
+set(IPHONEOS_ARCHS armv7;armv7s;arm64)
```

Comment:
32 bit architectures dropped since iOS 11.0:
* https://github.com/ruslo/polly/blob/2f1b497df080164f459a7733069267e7eafff03c/ios-11-0.cmake#L33-L34

Reply:
I think it's still valid to build 32-bit, as long as we're targeting <= iOS10 (but can still build with newer SDK)

> 32-bit Apps
> Deprecations
> 32-bit apps no longer run on iOS 11.
> To include a 32-bit slice in your app, target iOS 10 or earlier.

https://developer.apple.com/library/content/releasenotes/General/RN-iOSSDK-11/index.html